e3sm_diags/driver/lat_lon_driver.py

```diff
@@ -17,17 +16,18 @@
 
 def create_metrics(ref, test, ref_regrid, test_regrid, diff):
     """Creates the mean, max, min, rmse, corr in a dictionary"""
+    missing_value = 999.999
```

Comment:
It is a good idea to add a comment explaining why `missing_value` is instantiated to 999.999.

Reply:
The missing value 999.999 is used in another application of e3sm_diags for cmip model comparison by Chris, so I just went ahead and adopted it. There might be other alternatives, i.e. use NAN, but that will require more code change downstream.